### ISSUE TITLE
bytecode to native compiler for different processors

### ISSUE THREAD

**Issue by sPyOpenSource** [sPyOpenSource/compiler#34]
working in progress:
- [x] x86 processors
- [ ] [arm processors](https://en.wikipedia.org/wiki/ARM_architecture_family)

**Comment by sPyOpenSource** [comment]
I've created a class for arm processors in the compiler.

**Comment by sPyOpenSource** [comment]
[llvm](https://llvm.org) maybe the solution. we can translate the bytecode to ir

**Comment by jserv** [comment]
> [llvm](https://llvm.org) maybe the solution. translate the bytecode to ir

How about using [java2llvm](https://github.com/digitalgust/java2llvm)?

**Comment by sPyOpenSource** [comment]
yeah, I have already look in to java2llvm project. That is also what I'm thinking of.

**Comment by sPyOpenSource** [comment]
[SJP](https://fam-frenz.de/stefan/compiler.html) - a compiler that compiles a tailored subset of Java directly to native code, eliminating the need for bytecode and virtual machines.
